### DISCUSSION TITLE
Symbol Characters

### DISCUSSION THREAD

**Discussion by KyllianGamer** [Minestom/Minestom#1446] [Help] [answered]
So when I for example try to send a message to a player with following characters: "« ∞ »" I get the following result:

![afbeelding](https://user-images.githubusercontent.com/42489040/195907209-ccbec1dc-75ce-444d-b81f-2d4f7afda555.png)
Any way I can fix this?

**Comment by iam4722202468** [comment]
That is a unicode character, specifically U+221E. Unicode characters are more than one byte. The client reading it doesn't recognize the character as unicode and is reading it as ascii, meaning it thinks each byte is one character. If you need that character, you can add it to the resource pack.

**Reply by KyllianGamer** [reply]
Alright but it seems that when I replace the actual symbol itself with "\u221e" it does seem to display it correctly on the client side, so replacing them with those values should mean I don't need a resource pack.

**Reply by iam4722202468** [reply]
oh, cool

**Comment by Moulberry** [comment, marked as answer]
You most likely are compiling without UTF-8 support

Try adding the following to your build.gradle:
```
compileJava.options.encoding = 'UTF-8'

tasks.withType(JavaCompile) {
    options.encoding = 'UTF-8'
}
```

(From https://stackoverflow.com/questions/21267234/show-utf-8-text-properly-in-gradle)

**Reply by KyllianGamer** [reply]
Yep, thanks.
